Sage/carbon
# docs: update carbon public roadmap

### Proposed behaviour

- Update the main Carbon roadmap
- Update the installation and usage documentation
- Update `CarbonProvider`'s documentation
   - Create new stories for showcasing setting different themes with the provider.
   - Update docs to be in line with our current recommendation of using the `sageTheme` for projects going forward over legacy themes like `mintTheme`.
   - Amend accessibility cypress tests to check these new stories
- Add descriptions to `CarbonProvider` props 

### Current behaviour

- Public roadmap contains little information about our next long-term priorities
- `CarbonProvider` docs is unambiguous in places and contains little context about the difference between the modern approach of using design tokens over legacy themes.

### Checklist




#### QA


### Testing instructions

- Ensure added documentation wording is concise and non-ambiguous
- Ensure documentation content is up to date with modern Carbon functionality 
- Ensure all links in updated pages are working properly
